src/main/java/com/fundynamic/d2tm/game/entities/units/UnitsRepository.java

```diff
@@ -0,0 +1,47 @@
+package com.fundynamic.d2tm.game.entities.units;
+
+
+import com.fundynamic.d2tm.game.map.Map;
+import com.fundynamic.d2tm.math.Vector2D;
+import org.newdawn.slick.Image;
+import org.newdawn.slick.SlickException;
+
+public class UnitsRepository {
```

Comment:
Due to conventions, I prefer singular terms here. So `UnitRepository` and `StructureRepository`. Objections?